### DISCUSSION TITLE
Example for executing seeds?

### DISCUSSION THREAD

**Discussion by dyaacov** [sequelize/umzug#162] [Q&A] [answered]
Hi,
Can you please post an example of how to run migrations and how to run seeds?

thanks

**Comment by jagged3dge** [comment]
This might help:  https://github.com/sequelize/umzug/issues/24#issuecomment-591313032

**Comment by mmkal** [comment, marked as answer]
@dyaacov there's now an example with seeders: https://github.com/sequelize/umzug/tree/master/examples/4.sequelize-seeders

In this example there's no complex logic where seeders interleave with regular migrations as mentioned on the thread.

I imagine you found a solution since posting this, but feel free to comment here if you feel there's more that should be covered. CC the 👍 ers @voravor @robinsonaaron @franz-see @pickfire @lucasklassen @AndresERojasi @johnfdhartman @ealbuquerque @sarangnx
